docs/source/contents/4_draft_mapping.rst

```diff
@@ -2152,6 +2152,622 @@ Drop all values in mods:part.
+^^^^^
+:code:`relatedItem[@type='host'][@displayLabel='Bibliographic Citation']`
+
+Decision
+^^^^^^^^
+The `dcterms:bibliographicCitation` predicate was selected for these values.
+
+`Example record - arrow:1 <https://digital.lib.utk.edu/collections/islandora/object/arrow:1/datastream/MODS/view>`_
+
+.. code-block:: xml
+
+    <relatedItem type="host" displayLabel="Project">
+      <titleInfo>
+        <title>From Pi Beta Phi to Arrowmont</title>
+      </titleInfo>
+    </relatedItem>
+    <relatedItem displayLabel="Project Part" type="host">
+      <titleInfo>
+        <title>The Arrow of Pi Beta Phi</title>
+      </titleInfo>
+    </relatedItem>
+    <relatedItem displayLabel="Bibliographic Citation" type="host">
+      <titleInfo>
+        <title>The Arrow, Volume 27, Number 1</title>
+      </titleInfo>
+    </relatedItem>
+
+.. code-block:: turtle
+
+    @prefix dcterms: <http://purl.org/dc/terms/> .
+
+    <https://example.org/objects/1> dcterms:bibliographicCitation "The Arrow, Volume 27, Number 1" .
+
+relatedItem[@type="otherVersion"]
+---------------------------------
+Use Case
+^^^^^^^^
+`relatedItem[@type="otherVersion"]` is used to indicate identifying information about another version of the resource. It appears in the Van Vactor and Arrowmont metadata; it is used, respectively, to identify an alternate version of the sheet music or the scrapbook that holds the image.
+
+Justification
+^^^^^^^^^^^^^
+
+XPath
+^^^^^
+:code:`relatedItem[@type='otherVersion']/identifier` OR
+:code:`relatedItem[@type='otherVersion']/location/url`
+
+Decision
+^^^^^^^^
+See the following section on `relatedItem/identifier[@type]`.
+
+I'm not sure about the relevance of `relatedItem[@type='otherVersion']/location/url`. In theory, it would look something like this, but it would need to be added to our metadata/RDF after migration:
+
+`Example record - arrpgimg:319 <https://digital.lib.utk.edu/collections/islandora/object/arrpgimg:319/datastream/MODS/view>`_
+
+.. code-block:: xml
+
+    <relatedItem type="otherVersion">
+      <titleInfo>
+         <title>Gaitlinburg from All Sides</title>
+      </titleInfo>
+      <location>
+         <url>https://digital.lib.utk.edu/collections/islandora/object/arrowmont%3A16</url>
+      </location>
+    </relatedItem>
+
+.. code-block:: turtle
+
+    @prefix dcterms: <http://purl.org/dc/terms/> .
+
+    <https://example.org/objects/1> dcterms:hasVersion <uri-of-source-scrapbook-in-new-system> .
+
+relatedItem/identifier[@type]
+-----------------------------
+Use Case
+^^^^^^^^
+This XPath's `@type` has three distinct values: `local`, `catalog`, and `pid`. `@type='pid'` is used in collection-level records to indicate featured items and should not be migrated. `@type='local'` is used to indicate the manuscript number of the resource's archival collection. `@type='catalog'` is used in the Van Vactor collection to indicate the identifying number for an alternate version of the score.
+
+Justification
+^^^^^^^^^^^^^
```

Comment:
Some justification is needed - I don't think this is easy to do when you have three different use cases in this single category.

Reply:
Hopefully splitting these into subsections will be helpful. As has been said repeatedly on this PR - thank you for your thoughts!